DataFormats/Portable/interface/PortableHostCollectionReadRules.h

```diff
@@ -0,0 +1,76 @@
+#ifndef DataFormats_Portable_interface_PortableHostCollectionReadRules_h
+#define DataFormats_Portable_interface_PortableHostCollectionReadRules_h
+
+#include <TGenericClassInfo.h>
+#include <TVirtualObject.h>
+
+#include "DataFormats/Portable/interface/PortableHostCollection.h"
+#include "FWCore/Utilities/interface/concatenate.h"
+#include "FWCore/Utilities/interface/stringize.h"
```

Comment:
Could you add the explicit dependence on `FWCore/Utilities` to `DataFormats/Portable/BuildFile.xml`?

In principle an explicit dependence on `rootcling` (I believe) would be good as well.